storybooknative/babel.config.js.bak

```diff
@@ -0,0 +1,10 @@
+// @flow
```

Comment:
This is necessary as becomes clear when looking at `storybooknative/package.json`